postgres/assets/configuration/spec.yaml

```diff
@@ -282,6 +282,14 @@ files:
       value:
         type: boolean
         example: true
+    - name: collect_column_metrics
```

Comment:
Shall we add a parameter for the collection interval? The query can be slow, so we need a way to control frequency and reduce load on both the database and the agent.

Reply:
A good point - I overindexed on seeing `pg_stats` referenced in other metrics checks and assumed it was load we currently handle at the metrics interval (but that has different context as well)

I now see why you had mentioned including this as a part of schema check (at the 10?/20? minute interval); an interval here could be good too, but I also don't want to over-configure this if we can. Ideally if we could put it behind an existing functionality, that'd be good (like schema?)

Reply:
Schema collection has performance problems. For this reason, it isn't enabled by default. Collecting column statistics too often is likely to cause harm with benefit being only marginal. Exposing unnecessary config options to users is definitely a problem. A trade off is to introduce the config option for interval but make it hidden, if there are customers who would like to collect more aggressively can be instructed to adjust the parameter.